reframe/utility/sanity.py

```diff
@@ -514,7 +514,8 @@ def calc_bound(thres):
     try:
         evaluate(assert_bounded(val, lower, upper))
     except SanityError:
-        error_msg = '{0} is beyond reference value {1} (l={2}, u={3})'
+        error_msg = '%s ' % msg if msg else ''
+        error_msg += '{0} is beyond reference value {1} (l={2}, u={3})'
```

Comment:
This violates the purpose of `msg`. `msg` allows the user to override completely the default message. This is not the case here.

Reply:
I fixed it.